pkgs/development/python-modules/indexed_gzip/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pkgs
+, cython
+, pytest-runner
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "indexed_gzip";
+  version = "1.5.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "08v3j34vii2ad8v2clrw63dxyqxkccfaxfdv5qwbq5bny1dxrc7x";
+  };
+
+  buildInputs = [ cython pkgs.zlib ];
+
+  checkInputs = [ cython pytest-runner numpy ];
+
+  doCheck = false; # unclear cause of errors
+
```

Comment:
If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.